package-lock.json

```diff
@@ -10514,12 +10514,6 @@
       "integrity": "sha1-wguW2MYXdIqvHBYCF2DNJ/y4y3U=",
```

Comment:
do not commit the package-lock.json

we already have yarn.lock here

Reply:
ok